tests/test_strategy_infinite_bus.py

```diff
@@ -306,30 +306,88 @@ def testing_event_market_cycle_post_offers(bus_test3, area_test3):
 
 
 @pytest.fixture()
-def area_test4():
-    return FakeArea(0)
-
-
-@pytest.fixture()
-def bus_test4(area_test4):
+def bus_test4(area_test1):
     c = InfiniteBusStrategy(energy_sell_rate=30, energy_buy_rate=25)
-    c.area = area_test4
-    c.owner = area_test4
+    c.area = area_test1
+    c.owner = area_test1
     return c
 
 
-def testing_event_tick_buy_energy(bus_test4, area_test4):
+def testing_event_tick_buy_energy(bus_test4, area_test1):
     bus_test4.event_activate()
     bus_test4.event_tick()
-    assert len(area_test4.test_market.traded_offers) == 1
-    assert area_test4.test_market.traded_offers[-1].offer.energy == 1
+    assert len(area_test1.test_market.traded_offers) == 1
+    assert area_test1.test_market.traded_offers[-1].offer.energy == 1
 
 
-def testing_event_market_cycle_posting_bids(bus_test4, area_test4):
+def testing_event_market_cycle_posting_bids(bus_test4, area_test1):
     ConstSettings.IAASettings.MARKET_TYPE = 2
     bus_test4.event_activate()
     bus_test4.event_market_cycle()
     assert len(bus_test4._bids) == 1
-    assert bus_test4._bids[area_test4.test_market.id][-1].energy == sys.maxsize
-    assert bus_test4._bids[area_test4.test_market.id][-1].price == 25 * sys.maxsize
+    assert bus_test4._bids[area_test1.test_market.id][-1].energy == sys.maxsize
+    assert bus_test4._bids[area_test1.test_market.id][-1].price == 25 * sys.maxsize
     ConstSettings.IAASettings.MARKET_TYPE = 1
+
+
+def test_global_market_maker_rate_single_value(bus_test4):
+    assert isinstance(GlobalConfig.market_maker_rate, int)
+    assert (GlobalConfig.market_maker_rate ==
+            ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE)
+
+
+"""TEST5"""
```

Comment:
Why are these needed?

Reply:
Still following the convention in continuity to old test

Reply:
Yeah. I think at some point we can get rid of them. They don't seem to have any use.
Leave it here though.